app/controllers/api/v1/admin/server_rooms_controller.rb

```diff
@@ -6,18 +6,22 @@ module Admin
       class ServerRoomsController < ApiController
         before_action :find_server_room, only: :destroy
 
+        # GET /api/v1/admin/server_rooms.json
         def index
-          rooms = Room.all.search(params[:search])
-
-          rooms.map! do |room|
-            {
-              friendly_id: room.friendly_id,
-              name: room.name,
-              owner: User.find(room.user_id).name
-            }
+          rooms = Room.includes(:user).search(params[:search])
+          active_rooms = BigBlueButtonApi.new.active_meetings
+          active_rooms_hash = {}
+
+          active_rooms.each do |active_room|
+            active_rooms_hash[active_room[:meetingID]] = active_room[:participantCount]
+          end
```

Comment:
The conversion of the `active_rooms` array into an `active_rooms_hash` with the `meetingID` as key is good because it makes the subsequent matching operations much faster as arrays are not efficient for searching.